client/components/fields/editor/ScheduleHour.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import {superdeskApi} from '../../../superdeskApi';
+import {IEditorFieldProps} from '../../../interfaces';
+
+import {EditorFieldSelect} from './base/select';
+
+export class EditorFieldScheduleHour extends React.PureComponent<IEditorFieldProps> {
+    render() {
+        const {gettext} = superdeskApi.localization;
+        const onChange = (field: string, value: string) => {
+            this.props.onChange(field, parseInt(value, 10));
+        };
+
+        return (
+            <EditorFieldSelect
+                field={this.props.field ?? 'hour'}
+                label={this.props.label ?? gettext('Hour')}
+                labelField="label"
+                keyField="value"
+                valueAsString={true}
+                defaultValue={-1}
+                options={[
+                    {value: -1, label: gettext('Every Hour')},
```

Comment:
actually not sure if we should do this via gettext, might be better to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString